codegen/projections/white_label/spec/types_spec.rb

```diff
@@ -51,10 +51,10 @@ module Types
       it 'implements to_h for delegation' do
         string_union = Union::String.new('simple string')
         struct_union = Union::Struct.new(struct)
-        unknown_union = Union::Unknown.new({ key: 'key', value: 'value' })
+        unknown_union = Union::Unknown.new({ name: 'name', value: 'value' })
```

Comment:
Key was wrong from the POV of how our code actually creates these (we always create them using name and value) - but nothing in the generated Unknown class actually restricts the data that it holds.  

We could change how we generate the `Unknown` union member.  Right now it is:

```ruby
      class Unknown < Union
        def to_h
          { unknown: super(__getobj__) }
        end

        def to_s
          "#<WhiteLabel::Types::Unknown #{__getobj__ || 'nil'}>"
        end
      end
      
      # and the Hearth::Union is just a delegator
module Hearth
  # Top level class for all Union types
  class Union < ::SimpleDelegator
    include Hearth::Structure
  end
end
```

Instead of just allowing whatever data, we could write a class that intializes with kwargs that only allow name and value?

Reply:
Updated Union Unknown to constraint to name/value only.